src/edfi-paging-test/edfi_paging_test/api/request_client.py

```diff
@@ -61,6 +62,11 @@ def __init__(self, args: MainArguments) -> None:
         client = BackendApplicationClient(client_id=args.key)
         self.oauth = OAuth2Session(client=client)
         self.api_info : Optional[APIInfo] = None
+        # configure connection pool
+        requests_adapter = adapters.HTTPAdapter(
+            pool_connections=args.connectionLimit, pool_maxsize=args.connectionLimit
+        )
+        self.oauth.mount("https://", requests_adapter)
```

Comment:
Will this restrict someone from using the toolkit with plain `http`?

Reply:
Probably. I'll add http.